cmd/metamorph.go

```diff
@@ -309,13 +319,47 @@ func StartMetamorph(logger utils.Logger) (func(), error) {
 
 	return func() {
 		logger.Infof("Shutting down metamorph store")
+		metamorphProcessor.Shutdown()
 		err = s.Close(context.Background())
 		if err != nil {
 			logger.Errorf("Could not close store: %v", err)
 		}
 	}, nil
 }
 
+func NewStore(dbMode string, folder string) (s store.MetamorphStore, err error) {
```

Comment:
it's not quite obvious that default storage type is SQL here. Maybe worth keeping all the options as an argument instead?
Or document each of existing dbModes?

Reply:
Will revisit that